packages/@aws-cdk/aws-fsx/lib/lustre-file-system.ts

```diff
@@ -0,0 +1,305 @@
+import {Connections, ISubnet, Peer, Port, SecurityGroup} from '@aws-cdk/aws-ec2';
+import {Construct} from '@aws-cdk/core';
+import {FsxFileSystemAttributes, FsxFileSystemBase, FsxFileSystemProps, IFsxFileSystem} from './file-system';
+import {CfnFileSystem} from './fsx.generated';
+
+/**
+ * The different kinds of file system deployments used by Lustre.
+ */
+export enum FsxLustreDeploymentType {
+  /**
+   * Original type for shorter term data processing. Data is not replicated and does not persist on server fail.
+   */
+  SCRATCH_1 = 'SCRATCH_1',
+  /**
+   * Newer type for shorter term data processing. Data is not replicated and does not persist on server fail.
+   * Provides better support for spiky workloads.
+   */
+  SCRATCH_2 = 'SCRATCH_2',
+  /**
+   * Long term storage. Data is replicated and file servers are replaced if they fail.
+   */
+  PERSISTENT_1 = 'PERSISTENT_1'
+}
+
+/**
+ * The configuration for the Amazon FSx for Lustre file system.
+ *
+ * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-fsx-filesystem-lustreconfiguration.html
+ */
+export interface LustreConfiguration {
+  /**
+   * The type of backing file system deployment used by FSx.
+   */
+  readonly deploymentType: FsxLustreDeploymentType;
```

Comment:
Can we make a good guess at a default users are likely to want here? 

If the answer is "no", that is fine.

Reply:
When using the AWS Portal UI for creating a Lustre file system, it does default to `PERSISTENT_1`.  This gets tricky to mimic though, because that type then requires `perUnitStorageThroughput` to be set as well. This other property in the LustreConfiguration is only used with `PERSISTENT_1`, so if the user switched away from that, the default on `perUnitStorageThroughput` would need to be cleared.